optimum/onnxruntime/modeling_ort.py

```diff
@@ -730,3 +733,87 @@ def _prepare_attention_mask_for_generation(
         else:
             # Ensure attention mask is on the same device as the input IDs
             return torch.ones(inputs.shape[:2], dtype=torch.long, device=inputs.device)
+
+
+IMAGE_CLASSIFICATION_SAMPLE = r"""
```

Comment:
A bit nitpicking, but isn't it an "example" more than a "sample"?

Reply:
I fully agree with you, I just copy-pasted this from other similar classes XD
I'm going to push a new commit to modify this :)

Reply:
The naming is coming from transformers